packages/ui-components/src/components/toggle-tip/toggle-tip.tsx

```diff
@@ -74,11 +76,12 @@ export class KvToggleTip implements IToggleTip, IToggleTipEvents {
 	};
 
 	private onButtonClick = (ev: MouseEvent) => {
-		ev.stopPropagation();
+		if (this.blockElementClickEventPropagation) ev.stopPropagation();
```

Comment:
IMO, this should be the default behaviour. 
@ricardogoncalves89 any reason why we were stopping this event propagation? Does this have something to do with this being rendered inside a portal?